sway-fmt-v2/src/items/item_struct.rs

```diff
@@ -187,9 +187,11 @@ fn format_struct(
 
 impl ItemLen for ItemStruct {
     fn get_formatted_len(&self) -> Result<usize, FormatterError> {
```

Comment:
Since this function is the same for each of its use cases, would you mind making it simply a `pub(crate)` function?

Reply:
I did follow the trait path because of two reasons:

1. To handle all of them with a single function I need to take `ItemKind` as a function parameter and then match for struct, enum, and storage cases to call the respective format functions. This function can falsely imply that it will also return the lengths of other kinds of `ItemKind`. 
2. We may want to use other implementations later on for other `ItemKinds`.

That being said, I can also use a single function for this purpose and document it (regarding the 1st point), if you think that should be the way to go.